apps/docs/docs/state/getting-started.mdx

```diff
@@ -449,20 +461,117 @@ export class MovieListComponent {
 }
 ```
 
-## Custom state accumulation (mutability)
+## Configuration
+
+There are a couple of settings you can adjust when using `RxState`.
+
+### provideRxStateConfig
+
+Configurations for `RxState` instances are provided in the DI tree by using the `provideRxStateConfig` provider function.
+
+```typescript title="main.ts"
+import { bootstrapApplication } from '@angular/platform-browser';
+import { AppComponent } from './app.component';
+import { provideRxStateConfig } from '@rx-angular/state';
+
+bootstrapApplication(AppComponent, {
+  providers: [
+    provideRxStateConfig(),
+    /* define features here */
+  ],
+});
+```
+
+### Scheduler
+
+By default, `RxState` observes changes and computes new states on the [`queueScheduler`](https://rxjs.dev/api/index/const/queueScheduler). You can modify
+this behavior by using the `withScheduler()` or `withSyncScheduler()` configuration features.
+
+The most applicable use case will be to have sync scheduling:
```

Comment:
It's not clear to me when / why we would need to use sync scheduling. I would suggest to provide more guidance here.